pages/environments.md

```diff
@@ -24,3 +25,9 @@ Usually you'll need at least four environments, each servering a different purpo
     * Production also has a well defined process for deploying new test builds that have been promoted. See the [Build Promotion] ({{site.baseurl}}/build-promotion/) section.
 
 Note: Most (if not all) environments will need the full set of developer tools. Also, each environment should be sandboxed from each other to prevent resources from accidents.
+
+####Monitoring
+Applications mysteriously failing? Disks have filled to capacity for the 100th time? Everything is working...just VERY SLOWLY? Etc. There is a whole host of problems (bugs) that are difficult or impossible to find without good comprehensive monitoring of the environments. 
+A good starting point would be O'Reily's [The Monitoring Infrastructure Is Critical](http://sites.oreilly.com/odewahn/dds-field-guide/ch07.html) which lists lots of tools that can help with this ongoing problem.
+
+<a name="footnote1">1</a>: See [The Entire Application Runs Locally in Development](http://sites.oreilly.com/odewahn/dds-field-guide/ch05.html) from O'Reily
```

Comment:
We've recently voted against using footnotes in other documents. Can you promote this to appear inline?